src/analysis/interpretation/interpret.ts

```diff
@@ -76,6 +81,33 @@ function interpretPayloadChecks(
   };
 }
 
+function hasWarningsIgnorePatterns(warnings?: IgnoreWarningsPatterns): boolean {
+  return warnings !== undefined && Object.keys(warnings).length > 0;
+}
+
+export function excludeIgnoredDependenciesWarnings(
+  dependenciesWarnings: DependencyWarning[],
+  ignorePatterns: IgnorePatterns
+): DependencyWarning[] {
+  if (!hasWarningsIgnorePatterns(ignorePatterns?.warnings)) {
+    return dependenciesWarnings;
+  }
+
+  return dependenciesWarnings.filter(function excludeIgnorableWarnings(
+    dependencyWarnings
+  ) {
+    if (
+      dependencyWarnings.warnings.find((w) =>
```

Comment:
`w` ? seems a bit short to me ^^.

Reply:
Yeah, will definitely don't let it be like that.